index.html

```diff
@@ -32,16 +44,26 @@ <h1>Awesome Books</h1>
                     <td></td>
                 </tr>
             </table>
-        
-        </div>
-        <div class="book-form">
+        </section>
+        <section class="book-form" id="addBookForm">
+            <h2>Add More Books</h2>
             <form id="library">
                 <input type="text" id="title" placeholder="Title" required><br>
                 <input type="text" id="author" placeholder="Author" required><br>
                 <button type="button" id="addBookButton">Add Book</button>
-            </form>
-        </div>
+            </form>   
+        </section>
+        <section id="Contact-info">
+            <h2>Contact Information</h2>
+            <p>Do you have any questions or you just want to say "Hello"?<br>You can reach out to us!</p>
+            <ul>
+                <li>Our email: mail@mail.com</li>
+                <li>Our phone number: 0045625656565456</li>
+                <li>Our address: Streetname 22, 84503 City, Country</li>
+            </ul>
+        </section>
     </div>
+  </main>  
```

Comment:
[OPTIONAL] I think we could add a footer to the webpage indicating the copyright and information about it as shown in the wireframes to the bottom of our page.

For reference:
![image](https://user-images.githubusercontent.com/79114233/143723672-2a9cef3e-af2e-4649-8a6a-02ad007ebb71.png)